src/views/set2/Set2.vue

```diff
@@ -16216,131 +16342,131 @@
           self.myGroups_c2p2 = self.d3.map(csv_matrix_annual, function(d){return d[self.timestep_c2p2];}).keys()
           self.myVars_c2p2 = self.d3.map(csv_matrix_annual, function(d){return d.seg_id_nat;}).keys()
 
-          // build x scale for matrix cells
-          let x = self.d3.scaleBand()
-              .range([0, self.matrix_width_c2])
-              .domain(self.myGroups_c2p2)
-              .padding(0.1);
+          // // build x scale for matrix cells
```

Comment:
A huge chunk below this was part of the matrix building before it was prebuilt. Do you want to delete instead of just comment out? There will still be a record of it in Git.